DTM-Added map widget fix. (#1776)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1019-1-8-map-marker-visibility-based-on-location-property-values-in-map-widget.md

---
date:

> title: Map marker visibility based on Location property values in map widget

Reviewer: ```suggestion
title: Markers correctly displayed in the Map widget
```

product_area: Application enablement & solutions
change_type:
  - value: change-2c7RdTdXo4
    label: Fix
component:
  - value: component-Tl88RYb4A
    label: Digital Twin Manager
build_artifact:
  - value: tc-wYIY0MBDO
    label: dtm
ticket: IOT-19589
version: 1019.1.8
---

> When assets with location were imported in bulk in the "Map" widget, markers were not shown due to an incompatible data type of latitude and longitude values. Aligning the data type to number now correctly displays the markers on the map.

Reviewer 2: @BeateRixen, In this case, the assets are imported in bulk from DTM but the markers were not shown when viewing from Map widget. Kindly suggest a better way of putting this info as I think that 'When assets with location were imported in bulk in the "Map" widget' gives an impression that the assets are imported in bulk in the Map widget.

Reviewer: ```suggestion
When assets with location were imported in bulk, markers in the "Map" widget were not shown due to an incompatible data type of latitude and longitude values. Aligning the data type to number now correctly displays the markers on the map.
```

> In the Maps widget, for assets with locations when imported in bulk, markers were not shown due to incompatible data type of Latitude and Longitude values. Aligning the data type to number now correctly displays the markers on the map.

Reviewer: ```suggestion
When assets with location were imported in bulk in the "Map" widget, markers were not shown due to an incompatible data type of latitude and longitude values. Aligning the data type to number now correctly displays the markers on the map.
```
